newrelic/resource_newrelic_cloud_azure_integrations_test.go

```diff
@@ -247,6 +247,11 @@ resource "newrelic_cloud_azure_integrations" "bar" {
     resource_groups          = ["beyond"]
   }
 
+  service_fabric {
```

Comment:
`Service Fabric` is deprecated by Microsoft so we chose not to include it in the resource. Let's remove it from this test.